### ISSUE TITLE
cube file

### ISSUE THREAD

**Issue by darkzbaron** [semchan/NLUT#8]
Any chance I can save a cube file from the LUT computed by the model?

**Comment by semchan** [comment]
> Any chance I can save a cube file from the LUT computed by the model?

yes, you can. you can refer the code from: `def get_lut(opt, original, example):` But for useage, it may a little bit different from some commercial software, you should refer `inference_finetuning_video.py` or` inference_finetuning_image.py` for usage. thanks
